certification/internal/shell/operatorsdk.go

```diff
@@ -0,0 +1,45 @@
+package shell
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+)
+
+type OperatorSdkCLIEngine struct{}
+
+func (osce OperatorSdkCLIEngine) Scorecard(image string, opts cli.OperatorSdkScorecardOptions) (*cli.OperatorSdkScorecardReport, error) {
+	cmdArgs := []string{"scorecard", "--output"}
+	if opts.OutputFormat == "" {
+		opts.OutputFormat = "json"
+	}
+	cmdArgs = append(cmdArgs, opts.OutputFormat)
```

Comment:
Can we append the output format as you do on L24 instead of having `--output` be a part of the original `cmdArgs` value?